clients/go/test/grpc-app/Dockerfile

```diff
@@ -26,6 +26,8 @@ COPY --from=builder /go/bin/app /app
 
 # Copy the client config.
 COPY clients/go/test/grpc-app/config.yaml /etc/lumberjack/config.yaml
+COPY integration/testrunner/private_key.json /etc/lumberjack/private_key.json
```

Comment:
This is not needed for all test apps.

Reply:
Oh right, I forgot to remove it. Just removed.